docs/en/src/openapi/quickstart.md

```diff
@@ -42,7 +42,12 @@ async fn main() -> Result<(), std::io::Error> {
     // Start the server and specify that the root path of the API is /api, and the path of Swagger UI is /
     poem::Server::new(listener)
         .await?
-        .run(Route::new().nest("/api", api_service).nest("/", ui))
+        .run(
+            Route::new()
+            .nest("/openapi.json", api_service.spec_endpoint())
```

Comment:
You can use `at` here, because `api_service.spec_endpoint` does not contain nested paths. 🙂

```
.at("/openapi.json", api_service.spec_endpoint())
```

Reply:
Oh interesting, I didn't know about `at`, I'll add that in and it'll serve as a hint to others as well!